runtime/testy/runtimeUtil.py

```diff
@@ -9,6 +9,7 @@ class BAD_EVENTS(Enum):
   STUDENT_CODE_TIMEOUT    = "Student Code Timed Out"
   UNKNOWN_PROCESS         = "Unknown State Manager process name"
   STATE_MANAGER_KEY_ERROR = "Error accessing key in State Manager"
+  END_EVENT               = "Process terminated" # Used for testing
```

Comment:
To what? its not exactly that the process ended gracefully either; or at least, it's only true in testing. An actual runtime should never send this event.